mulighetsrommet-api/src/main/resources/web/openapi.yaml

```diff
@@ -6114,6 +6161,15 @@ components:
           type: number
       required: [id, tilsagnId, belop]
 
+    DelutbetalingBulkRequest:
```

Comment:
Angående `Bulk`- (eller er det `Bolk`?) -requester så _tror_ jeg ikke dette er type navngiving vi har benyttet oss av hittil.

Lurer på om noe ala `OpprettDelutbetalingerRequest` kanskje minner litt om navngivingen som har blitt benyttet hittil i api'et (selv om vi neppe er superkonskevente her :sweat_smile:)?

Reply:
Hehe, enig i at den kan få et bedre navn! Kan gjerne gå for `OpprettDelutbetalingRequest `i stedet om den klinger bedre. 😄